backend/sky_lectures/urls.py

```diff
@@ -15,7 +15,25 @@
 """
 from django.contrib import admin
 from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.views import get_schema_view
+import sky_lectures.settings.base
```

Comment:
You should get settings like this:
```suggestion
from django.conf import settings
```